docs/sections/user_guide/cli/mode_rocoto.rst

```diff
@@ -48,7 +53,155 @@ The ``uw`` mode for realizing and validating Rocoto XML documents
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use a UW YAML file ``rocoto.yaml`` with content
+
+.. code:: python
+
+   workflow:
+     attrs:
+       realtime: false
+       scheduler: slurm
+     cycledef:
+       - attrs:
+           group: howdy
+         spec: 202209290000 202209300000 06:00:00
+     entities:
+       ACCOUNT: myaccount
+       FOO: test.log
+     log: /some/path/to/&FOO;
+     tasks:
+       task_hello:
+         attrs:
+           cycledefs: howdy
+         account: "&ACCOUNT;"
+         command: "echo hello $person"
+         jobname: hello
+         nodes: 1:ppn=1
+         walltime: 00:01:00
+         envars:
+           person: siri
+
+
+* Realize a UW YAML input file to ``stdout`` in Rocoto XML format:
+
+  .. code:: XML
+
+    $ uw rocoto realize --input-file rocoto.yaml
+    [2024-01-02T13:41:25]     INFO 0 schema-validation errors found
```

Comment:
Shouldn't this be `UW schema-validation errors`?

Reply:
I might have mentioned this before, but: At some point I'd like to see if Sphinx has support for inserting content in .rst files by running external commands. If so, then we could consider saving these bits of config and the commands that process them in a set of external files and having Sphinx populate the content by running `uw` commands on those files. This could also form the basis for a later set of integration tests, which could execute the same examples (plus potentially more) as provided in the documentation, thereby keeping the documentation honest, too.